public/app/plugins/datasource/loki/components/useLokiSyntax.test.ts

```diff
@@ -66,8 +66,6 @@ describe('useLokiSyntax hook', () => {
 
     act(() => result.current.setActiveOption([activeOptionMock]));
 
-    await waitForNextUpdate();
-
-    expect(result.current.logLabelOptions).toEqual(logLabelOptionsMock3);
+    setTimeout(() => expect(result.current.logLabelOptions).toEqual(logLabelOptionsMock3));
```

Comment:
@boyskila can you elaborate on this change? Can't see how this relates to dependencies update

Reply:
Just revert it. See my last commit